pkg/util/xorm/dialect_postgres.go

```diff
@@ -1096,9 +1096,8 @@ func (db *postgres) GetTables() ([]*core.Table, error) {
 }
 
 func getIndexColName(indexdef string) []string {
-	var colNames []string
-
 	cs := strings.Split(indexdef, "(")
+	colNames := make([]string, 0, len(strings.Split(strings.Split(cs[1], ")")[0], ",")))
```

Comment:
Does it make sense to allocate `strings.Split(strings.Split(cs[1], ")")[0], ",")` and then call len and range on it below rather than doing the operation twice?

Reply:
sure, it won't hurt. I generally (vaguely, this isn't a firm heuristic) only allocate a variable if it's used more than ~twice unless it's a very expensive call, and this isn't, but i can change it